backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/bpartner/service/impl/BPartnerOrgBL.java

```diff
@@ -149,4 +130,16 @@ public org.compiere.model.I_AD_User retrieveUserInChargeOrNull(final Properties
 		}
 		return defaultContact;
 	}
+
+	@Override
+	public Collection<UserId> retrieveOrgUsers(final OrgId orgId)
+	{
+		final IBPartnerDAO bpartnerDAO = Services.get(IBPartnerDAO.class);
```

Comment:
is there any reason for not making it a field? (cyclic dep?)

Reply:
loud thinking: in that case, i think it would be nice to add `// cyclic dependency` or something, to not be asked again by reviewer

Reply:
moved to field.